openapi-cli/src/test/java/io/ballerina/openapi/cmd/OpenAPICmdTest.java

```diff
@@ -1025,6 +1025,89 @@ public void testFlattenCmdWithComposedSchema() throws IOException {
+        Path expectedFilePath = resourceDir.resolve(Paths.get("cmd/sanitize/sanitized_openapi_expected_1.json"));
+        String[] args = {"-i", resourceDir + "/cmd/sanitize/openapi.yaml", "-o", tmpDir.toString(), "-f", "json"};
+        Sanitize sanitize = new Sanitize();
+        new CommandLine(sanitize).parseArgs(args);
+        sanitize.execute();
+        compareFiles(expectedFilePath, tmpDir.resolve("sanitized_openapi.json"));
+    }
+
+    @Test(description = "Test openapi sanitize sub command with the yaml format option")
+    public void testSanitizeCmdYamlFormat() throws IOException {
+        Path expectedFilePath = resourceDir.resolve(Paths.get("cmd/sanitize/sanitized_openapi_expected_1.yaml"));
+        String[] args = {"-i", resourceDir + "/cmd/sanitize/openapi.json", "-o", tmpDir.toString(), "-f", "yaml"};
+        Sanitize sanitize = new Sanitize();
+        new CommandLine(sanitize).parseArgs(args);
+        sanitize.execute();
+        compareFiles(expectedFilePath, tmpDir.resolve("sanitized_openapi.yaml"));
+    }
+
+    @Test(description = "Test openapi sanitize sub command with the tags option")
+    public void testSanitizeCmdTags() throws IOException {
+        Path expectedFilePath = resourceDir.resolve(
+                Paths.get("cmd/sanitize/sanitized_openapi_expected_albums.json"));
+        String[] args = {"-i", resourceDir + "/cmd/sanitize/openapi.json", "-o", tmpDir.toString(), "-t", "albums"};
+        Sanitize sanitize = new Sanitize();
+        new CommandLine(sanitize).parseArgs(args);
+        sanitize.execute();
+        compareFiles(expectedFilePath, tmpDir.resolve("sanitized_openapi.json"));
+    }
+
+    @Test(description = "Test openapi sanitize sub command with the operations option")
+    public void testSanitizeCmdOperations() throws IOException {
+        Path expectedFilePath = resourceDir.resolve(
+                Paths.get("cmd/sanitize/sanitized_openapi_expected_operations.json"));
+        String[] args = {"-i", resourceDir + "/cmd/sanitize/openapi.json", "-o", tmpDir.toString(), "--operations",
+                "getAlbumById,getAlbums"};
+        Sanitize sanitize = new Sanitize();
+        new CommandLine(sanitize).parseArgs(args);
+        sanitize.execute();
+        compareFiles(expectedFilePath, tmpDir.resolve("sanitized_openapi.json"));
+    }
+
+    @Test(description = "Test openapi sanitize sub command with composed schema")
+    public void testSanitizeCmdWithComposedSchema() throws IOException {
+        Path expectedFilePath = resourceDir.resolve(Paths.get("cmd/sanitize/sanitized_openapi_composed_schema.json"));
+        String[] args = {"-i", resourceDir + "/cmd/sanitize/openapi_composed_schema.json", "-o", tmpDir.toString()};
+        Sanitize sanitize = new Sanitize();
+        new CommandLine(sanitize).parseArgs(args);
+        sanitize.execute();
+        compareFiles(expectedFilePath, tmpDir.resolve("sanitized_openapi.json"));
+    }
+
```

Comment:
Shall we add tests for different OpenAPI versions, (v2, v3.0.0, v3.0.1)?

Reply:
Fixed via [4befa98](https://github.com/ballerina-platform/openapi-tools/pull/1770/commits/4befa98e26f9a3af611f1cce7327d236ff7bf30e)